system/inc/system_ble_prov.h

```diff
@@ -0,0 +1,23 @@
+
+#pragma once
+
+#ifdef __cplusplus
+extern "C" {
+#endif // __cplusplus
+
+#include "ble_hal.h"
+
+#if HAL_PLATFORM_BLE
+
+int system_ble_prov_mode(bool enabled, void* reserved);
```

Comment:
Nit: let's make sure all of these have a common prefix e.g. `system_ble_prov_xxx`